### DISCUSSION TITLE
Can't drop a component twice

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4070] [Q&A]
Hello!
I can't drop a component twice on canvas.

Here a working copy: https://jsfiddle.net/L63hbc57/

I don't know what i can do, maybe I missed something in the documentation?

**Comment by gustavohleal** [comment]
Hi
Same problem here.

**Comment by artf** [comment]
Remove the id from the block
```js
content: {
      type: 'FormBlock',
      id: 'Form', // remove this one
      class: 'dina-form',
    }
```
